### PR TITLE
Hamburger menu animation

### LINKED ISSUE #50: Hamburger menu animation

### PR DESCRIPTION
**Pull request by LouiaR**
Type: patch

Fixes #50

### CODE REVIEW

File: src/components/Molecules/Header/Burger/BurgerMenu.js [hunk 1]
```diff
@@ -2,29 +2,34 @@
...
-const BurgerMenu = ({ toggle }) => {
+const BurgerMenu = ({ toggle, isExpandable }) => {
   return (
-    <Wrapper
-      href="#!/Header/1"
+    <BurgerWrapper
+      href="#"
       role="button"
       aria-haspopup="true"
       aria-label="Open and close Navigation Menu, 24 items listed"
```
Comment (gusliedke) [inline src/components/Molecules/Header/Burger/BurgerMenu.js]: Remove "24 items listed"

Comment (LouiaR) [reply src/components/Molecules/Header/Burger/BurgerMenu.js]: Done

File: src/components/Molecules/Header/Burger/BurgerMenu.style.js [hunk 2]
```diff
@@ -4,55 +4,68 @@ import hideVisually from '../../../../theme/shared/hideVisually';
 import Link from '../../../Atoms/Link/Link';
 
 /**
- * Burger menu
+ * Burger hidden text
  */
-const BurgerMenu = styled(Link)`
+const BurgerHiddenText = styled.div`
```
Comment (gusliedke) [inline src/components/Molecules/Header/Burger/BurgerMenu.style.js]:
```suggestion
const BurgerHiddenText = styled(Text)`
${hideVisually}
`;
```

Comment (gusliedke) [reply src/components/Molecules/Header/Burger/BurgerMenu.style.js]: And remove `<Text>`

Comment (LouiaR) [reply src/components/Molecules/Header/Burger/BurgerMenu.style.js]: Done

**Pull request merged**